VideoWeb/VideoWeb/ClientApp/.browserslistrc

```diff
@@ -6,4 +6,4 @@
 last 2 versions
 Firefox ESR
 not dead
-IE 9-11
\ No newline at end of file
+# IE 9-11
```

Comment:
I believe this was uncommented to show the unsupported browser page on IE.

Reply:
We can still leave this line as is. The config is there to ensure the build works for older browsers. If we're only showing an unsupported page I don't think we need the whole app to be checked against IE.

I would be happy to revert this though

Reply:
We can display something to IE users without having the app actually work for them just to display an unsupported browser page all right. I'm not entirely clear why the change was made in the first place, I believe it was a local issue so I'm ok with this as is and I'll bring it up again else where.